### PR TITLE
feat(apple): Add visionOS as experimental

### PR DESCRIPTION
**Pull request by philipphofmann**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

The Cocoa SDK supports visionOS as an experimental platform. Therefore, we can also add it to the docs.

### CODE REVIEW

**Comment by shanamatthews** [comment]
@philipphofmann - huh, I have no idea why this build is failing. I'll take another look when I have a sec tomorrow 👀

**Comment by philipphofmann** [comment]
Me neither. I had some back and forth with Daniel to add me to Vercel, but it's not working yet. If you could have a look, that would be excellent @shanamatthews.

File: src/platform-includes/getting-started-primer/apple.mdx
Comment (shanamatthews) [inline src/platform-includes/getting-started-primer/apple.mdx:1]: This file isn't currently used, so this change doesn't show up.

I think the current pattern would be to create a src/platform-includes/getting-started-primer/apple.visionos.mdx file and put this note there.

Comment (philipphofmann) [reply src/platform-includes/getting-started-primer/apple.mdx:1]: I fixed it by creating a getting started-primer.mdx for every apple platform.

**Review by shanamatthews — APPROVED** [review]
Looks good now, thanks for fixing that!

**Pull request merged**